compound_disease/compound_treats_disease/edge_prediction_experiment/edge_prediction_experiment.py

```diff
@@ -0,0 +1,330 @@
+
+# coding: utf-8
+
+# # Compound Treats Disease Edge Prediction
+
+# This notebook is designed to take the next step moving from predicted sentences to edge predictions
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+import pandas as pd
+from sklearn.metrics import precision_recall_curve, roc_curve, auc
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+
+# In[2]:
+
+
+total_candidates_df = (
```

Comment:
Maybe add a short comment to describe what each df is.  I couldn't really tell from the name, I could only tell after looking at the output